src/routes/MapExplorer/MapExplorer.js

```diff
@@ -6,19 +6,23 @@ import combine from '@turf/combine'
 import { featureCollection, point } from '@turf/helpers'
 import React from 'react'
 import { FlyToInterpolator } from 'react-map-gl'
-import { debounce, groupBy, isEqual } from 'lodash'
+import { debounce, get, groupBy, isEqual } from 'lodash'
 import cx from 'classnames'
+import { generateViewParams } from 'util/savedSearch'
 import Icon from 'components/Icon'
 import Loading from 'components/Loading'
-import { FEATURE_TYPES } from './MapExplorer.store'
+import { FEATURE_TYPES, formatBoundingBox } from './MapExplorer.store'
 import Map from 'components/Map/Map'
 import MapDrawer from './MapDrawer'
+import SavedSearches from './SavedSearches'
 import { createIconLayerFromPostsAndMembers } from 'components/Map/layers/clusterLayer'
 import { createIconLayerFromCommunities } from 'components/Map/layers/iconLayer'
+import { heart } from 'util/assets'
```

Comment:
i agree would be better to put this in the icon font, so lets go over that together